README.md

```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
 
 Contributions to the osu!wiki are now done through GitHub.
 
-A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [here](/wiki/osu!wiki_contribution_guide).
+A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [Article Style Guide](/wiki/Article_Style_Guide)..
```

Comment:
Shouldn't you just merge lines 25 and 27 into one; something like "A basic guide to using Github for the osu!wiki project that _must_ be adhered to for all new and rewritten pages going forward is available here: [link]." Having two separate lines leading to the same folder seems redundant.